app/views/tweets/profile.html.haml

```diff
@@ -0,0 +1,32 @@
+= render 'layouts/nav'
+
+.container.mt-5
+
+  = button_tag "Share your Journey", class: "btn btn-primary mb-3", type: "button"
+
+  .card
+    .card-body
+      .media
+        = image_tag "https://picsum.photos/130/170", class: "align-self-start mr-3"
+        .media-body
+          %h5.mt-0 Top-aligned media
+          %p Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente quia aut dolor odio voluptatibus error ipsa alias magnam officiis accusantium, deleniti natus asperiores quibusdam ad fuga odit provident tempore sequi.
+
+          %small
+            .text-muted Following: 0
+          %small
+            .text-muted Followers: 0
+          %small
+            .text-muted Journeys: 0
+
+  .card-columns.mt-3
+    - @tweets.each do |tweet|
```

Comment:
Consider factoring this out into a `_tweet.html.haml` partial.
Then you can just `render @tweets`

Reply:
```
.card-columns.mt-3
  = render @tweets
```
_tweet.html.haml
```
.card
  .card-body
    %h5.card-title
      .badge.badge-primary
        = "@" + tweet.user.username
    %p.card-text
      = tweet.text
    %small.text-muted
      = tweet.created_at
```

Reply:
Transferring this to a separate partial file as you've mentioned caused the tweets to render thrice as expected.  

![Screenshot from 2019-07-05 18-02-32](https://user-images.githubusercontent.com/21337635/60715267-37dd6600-9f4f-11e9-8fba-361135c3988e.png)

```haml
-# index.html.haml

.card-columns.mt-3
    = render @tweets
```

```haml
-# _tweet.html.haml

- @tweets.each do |tweet|
  .card
    .card-body
      %h5.card-title
        .badge.badge-primary
          = "@" + tweet.user.username
      %p.card-text
        = tweet.text
      %small.text-muted
        = tweet.created_at
```

Reply:
Remove the `each` Nards, it's no longer needed. See my other comment below.

Reply:
Noted, thanks po!